src/Campaigns/Routes/CampaignOverviewStatistics.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Give\Campaigns\Routes;
+
+use DateInterval;
+use DatePeriod;
+use DateTimeImmutable;
+use Give\API\RestRoute;
+use Give\Campaigns\CampaignDonationQuery;
+use Give\Campaigns\Models\Campaign;
+use Give\Framework\Support\Facades\DateTime\Temporal;
+
+/**
+ * @unreleased
+ */
+class CampaignOverviewStatistics implements RestRoute
+{
+    /** @var string */
+    protected $endpoint = 'campaign-overview-statistics';
+
+    /**
+     * @unreleased
+     */
+    public function registerRoute()
+    {
+        register_rest_route(
+            'give-api/v2',
+            $this->endpoint,
+            [
+                [
+                    'methods' => 'GET',
```

Comment:
Let's use the `WP_REST_Server::READABLE` to keep the consistency with the other routes.

Reply:
Resolved, see [083c72a](https://github.com/impress-org/givewp/pull/7545/commits/083c72a119d173bfb5f8a43b6f4d598304c6d1a2).